examples/applications/plot_cyclical_feature_engineering.py

```diff
@@ -209,11 +209,12 @@
             ("categorical", ordinal_encoder, categorical_columns),
         ],
         remainder="passthrough",
+        verbose_feature_names_out=False,
```

Comment:
Maybe give a hint at why this is necessary?

Reply:
Done in 01d390d.